ports/open62541/vcpkg.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "open62541",
+  "version-string": "1.1.2",
```

```suggestion
  "version": "1.1.2",
```
This field might be `version `here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions